src/core.js

```diff
@@ -983,6 +983,28 @@ const _makeWindow = (options = {}, parent = null, top = null) => {
       return fakeVrDisplay;
     },
     getGamepads,
+    clipboard:{
+      read:() => Promise.resolve(), // Not implemented yet
+      readText: () => new Promise((resolve, reject) => {
+        if (GlobalContext.contexts && GlobalContext.contexts.length){
+          const windowHandle = GlobalContext.contexts[0].getWindowHandle();
+          const clipboardContents = nativeWindow.getClipboard(windowHandle).slice(0, 256);// why do we slice this?
+          resolve(clipboardContents);
+        }else{
+          reject();
```

Comment:
This would be a puzzling error to recieve.

Reply:
change to `reject(new Error('No context available.'));` instead?

Reply:
That would be ok, but this brings up the question of why a gl context is needed to read clipboard. Is there any way to do it without a window/canvas?

Reply:
Indeed! I have replaced the passing of the window context in favor of `GetCurrentWindowContext` which removes the need for the error condition at all.